docker/terra-node-zsh/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM cerner/terra-node-base:latest
```

Comment:
Can we just declare this as our dev env instead of zsh in the case we ever want to swap this?

Reply:
so rename this container `terra-node-dev`?